.github/workflows/ci.yaml

```diff
@@ -35,11 +35,8 @@ jobs:
       - name: Install dependencies
         run: poetry install --no-interaction
 
-      - name: Lint (flake8)
-        run: poetry run pre-commit run flake8 --all-files --show-diff-on-failure
-
-      - name: Order of imports (isort)
-        run: poetry run pre-commit run isort --all-files --show-diff-on-failure
+      - name: Lint (ruff)
+        run: poetry run pre-commit run ruff --all-files --show-diff-on-failure
 
       - name: Formatting (black)
         run: poetry run pre-commit run black --all-files --show-diff-on-failure
```

Comment:
what's your opinion on creating a separate branch to test `ruff format` so that we can see the diff? if it doesn't change between alpha and beta I'd say we can switch to it.

Reply:
Sounds good

Reply:
https://github.com/bakdata/kpops/pull/365